chpl/chpl-service/src/main/java/gov/healthit/chpl/CHPLServiceConfig.java

```diff
@@ -57,6 +55,12 @@
 public class CHPLServiceConfig extends WebMvcConfigurerAdapter implements EnvironmentAware {
 
     private static final Logger LOGGER = LogManager.getLogger(CHPLServiceConfig.class);
+    private static final int MAX_UPLOAD_SIZE_BYTES = 5242880; //5MB
+    private static final int MAX_COOKIE_AGE = 3600;
```

Comment:
Love comment above; request comment here... Is this 3600 seconds? Milliseconds? Other?